api_fhir/models/term.py

```diff
@@ -0,0 +1,7 @@
+from api_fhir.models import DomainResource, Property, BackboneElement
+
+
+class Term(BackboneElement):
+
+    identifier = Property('identifier', 'Identifier', count_max='*')
+    applies = Property('applies', 'Period')
```

Comment:
Please add the newline at the end of the file.

Reply:
good catch thx